specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2021-09-01/privateLinkScopes_API.json

```diff
@@ -0,0 +1,1199 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "ApplicationInsightsManagementClient",
+    "description": "Azure Application Insights API reference for Private Links Scopes management.",
+    "version": "2021-09-01"
```

Comment:
>2021-09-01 [](http://example.com/codeflow?start=16&length=10)

The title suggests 2022-08-23 is being introduced whereas the version number here is 2021-09-01. Which of these is the correct one?